### PR TITLE
[FEATURE] Display QC metrics of var #239

### PR DESCRIPTION
**Pull request by Imipenem**
**PR Checklist**

-   [x] This comment contains a description of changes (with reason)
-   [x] Referenced issue is linked
-   [x] Documentation in `docs` is updated

**Description of changes**

- display qc metrics of var with extra columns if wanted

- updated docs

@Zethson : are the QC metrics of var (the metric names) fixed or can they vary throughout different datasets? Cause currently the implementation just take every column in var and displays them, but that's not the desired behaviour!

### CODE REVIEW

**Comment by Zethson** [comment]
"are the QC metrics of var (the metric names) fixed or can they vary throughout different datasets? Cause currently the implementation just take every column in var and displays them, but that's not the desired behaviour!"

Fixed so far, but maybe add a parameter to add additional columns for the display because people can add custom ones?

**Comment by Imipenem** [comment]
> Fixed so far, but maybe add a parameter to add additional columns for the display because people can add custom ones?

Way to go for now, y

File: docs/usage/api.rst [hunk 1]
```diff
@@ -62,6 +62,7 @@ Quality control
     :toctree: preprocessing
 
     preprocessing.calculate_qc_metrics
+    preprocessing.display_qc_metrics
```
Comment (Zethson) [inline docs/usage/api.rst:65]: Should this be in the plotting part?

Maybe pp.qc_metrics and pl.qc_metrics?

**Review by Zethson — APPROVED** [review]
LGTM, but maybe we can move things around?

File: docs/usage/api.rst (hunk 1 above)
Comment (Zethson) [reply docs/usage/api.rst:65]: (I'd suggest to remove the "calculate" and "display" as well then.

e.g. umap is tl.umap and pl.umap

Must be reflected in the tutorials as well.

File: ehrapy/plot/ehrapy_plot/plot_qc.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from anndata import AnnData
+from rich.console import Console
+from rich.table import Table
+
+
+def qc_metrics(adata: AnnData, extra_columns: list[str] | None = None) -> None:
```
Comment (Zethson) [inline ehrapy/plot/ehrapy_plot/plot_qc.py:8]: Think we need to `# pragma: no cover` this function?

**Review by Zethson — APPROVED** [review]
LGTM

File: ehrapy/plot/ehrapy_plot/plot_qc.py (hunk 2 above)
Comment (Imipenem) [reply ehrapy/plot/ehrapy_plot/plot_qc.py:8]: Y, theres no proper way of testing this one.

**Pull request merged**